src/libs/actions/Wallet.js

```diff
@@ -377,6 +377,62 @@ function activateWallet(currentStep, parameters) {
         });
 }
 
+/**
+ * Creates an identity check by calling Onfido's API with data returned from the SDK
+ *
+ * The API will always return the updated userWallet in the response as a convenience so we can avoid an additional
+ * API request to fetch the userWallet after we call VerifyIdentity
+ *
+ * @param {Object} parameters
+ * @param {String} [parameters.onfidoData] - JSON string
+ */
+function verifyIdentity(parameters) {
+    const onfidoData = parameters.onfidoData;
+
+    API.write('VerifyIdentity', {
+        onfidoData,
+    }, {
+        optimisticData: [
+            {
+                onyxMethod: CONST.ONYX.METHOD.MERGE,
+                key: ONYXKEYS.WALLET_ONFIDO,
+                value: {
+                    loading: true,
+                    errors: {},
+                    fixableErrors: [],
```

Comment:
`errors` and `fixableErrors` should be set to `null` instead. I believe the default behavior of `lodash.merge()` will take any existing object and merge it with an empty object, but not replace with the object (and something similar for arrays).